sql/core/src/test/resources/sql-tests/inputs/group-by-filter.sql

```diff
@@ -0,0 +1,29 @@
+-- Test filter clause for aggregate expression.
+
+-- Test data.
+CREATE OR REPLACE TEMPORARY VIEW testData AS SELECT * FROM VALUES
```

Comment:
Can you write most of tests for this pr here where possible? If you cannot, plz use `SQLQuerySuite` instead. (Currently, I think some tests are overlapped between `group-by-filter.sql` and `SQLQuerySuite`.

Reply:
I will add tests in `group-by-filter.sql`.